interceptor-http-test/build.gradle.kts

```diff
@@ -79,7 +79,8 @@ kotlin {
         }
         val nativeAgentMain by getting {
             dependencies {
-                implementation(project(":logger"))
+                if(HostManager.hostIsMingw) implementation(project(":logging-native"))
```

Comment:
why `mingw` requires specific dep

Reply:
Hack for IDEA:
- `:logging-native` is a transitive dep
- IDEA "cannot see" that dep -> throws error

It does not affect build process